tests/test_om_prior.py

```diff
@@ -1,32 +1,175 @@
+import datetime
+import os
 import subprocess
+import sys
+from pathlib import Path
 
+import numpy as np
+import pandas as pd
 import pytest
+import requests
 import xarray as xr
-import os
+from openmethane_prior.layers.omGFASEmis import downloadGFAS
+from openmethane_prior.omInputs import livestockDataPath, sectoralEmissionsPath
+from openmethane_prior.omUtils import getenv, secsPerYear
+
+root_path = Path(__file__).parent.parent
+sys.path.insert(1, os.path.join(root_path, "scripts"))
+from omDownloadInputs import download_input_files, downloads, remote, sectoralEmissionsPath
+
+
+@pytest.fixture(scope="session")
+def output_domain_file(root_dir,
+                       input_files
+                       ) :
+    subprocess.run(["python", os.path.join(root_dir, "scripts/omCreateDomainInfo.py")], check=True)
+
+    subprocess.run(["python", os.path.join(root_dir, "scripts/omPrior.py"), "2022-07-01", "2022-07-02"], check=True)
+
+    # Generated by scripts/omCreateDomainInfo.py
+    filepath_in_domain = os.path.join(root_dir, "inputs/om-domain-info.nc")
+
+    # Generated by scripts/omPrior.py
+    filepath_out_domain = os.path.join(root_dir, "outputs/out-om-domain-info.nc")
+
+    yield xr.load_dataset(filepath_out_domain)
+
+    os.remove(filepath_out_domain)
+
+    os.remove(filepath_in_domain)
+
+
+# Fixture to download and later remove all input files
+@pytest.fixture(scope="session")
```

Comment:
This fixture can probably move into `conftest.py` as it could be used in other places. Also since it is session scoped even if you use it in multiple test modules it will still only run once